app/views/layouts/application.html.haml

```diff
@@ -1,8 +1,8 @@
 %html
-  %body
+  %head
     = render "customer/head"
-    %body.customer
-      =render "customer/top_nav"
-      =render "customer/content"
-      =render "customer/footer"
+  %body.customer
+    =render "customer/top_nav"
```

Comment:
Naming? Should have change the layout name, or move partials to another folder with appropriate name

Reply:
oke em